app/views/favorite_topics/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Favortie Topics</h1>
+<div class="container">
+  <h1>Favortie Topics</h1>
```

Comment:
Typo here

Reply:
this  typo should be resolved in the next merge